server/bed.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @fileoverview Bed data process
+ */
+
+'use strict';
+
+var fs = require('fs');
+
+module.exports = {
+
+  /**
+   * Reads a separated bed file for one chromosome.
+   * @param {string} bedFile Path to the bed file.
+   * @returns {Array} Contains the intervals of bed data.
+   */
+  readBed: function(bedFile, xl, xr) {
+    var data = [];
+    var lines = fs.readFileSync(bedFile).toString().split('\n');
+    for (var lineNum in lines) {
```

Comment:
forEach is preferred for iterating javascript arrays. 

Avoid iterating keys of arrays using for (var key in Array).